### ISSUE TITLE
Tornado growth

### ISSUE THREAD

**Issue by KlutchGTR** [Mrbt0907/Weather2-Remastered#106]
I am so sorry for asking again 😅, but how to make a tornado grow to EF5? And how to make it violent/neverdissipate/natural and firenado?

**Comment by ProfessorFartsalot** [comment]
I believe you do /storm create <type> <flags>

**Comment by ProfessorFartsalot** [comment]
Thank you, I knew I was missing something in my command example xD

**Comment by KlutchGTR** [comment]
Ahh i see..

**Comment by KlutchGTR** [comment]
So by just increasing the chances of EF5/EF4/EF3 then it can spawn?

**Comment by Mrbt0907** [comment]
Yes, but you also have to lower the chances of the other values so they spawn less.

**Comment by KlutchGTR** [comment]
Hmmm, so the other values (EF0-EF2) need to be lower so that ef3-5 can spawn?

**Comment by Mrbt0907** [comment]
If you lower the other chance values, you have a higher chance for the higher stages to be spawned.

**Comment by Mrbt0907** [comment]
I hope you got your issue resolved. I will mark this as closed.
